Source/MQTTnet.Server/Options/MqttServerTlsTcpEndpointOptions.cs

```diff
@@ -22,7 +22,7 @@ public MqttServerTlsTcpEndpointOptions()
 
         public bool CheckCertificateRevocation { get; set; }
 
-        public SslProtocols SslProtocol { get; set; } = SslProtocols.Tls12;
```

Comment:
Please add a summary comment including a reference to the MSDN so that users will understand the reason for "None" which looks like that SSL is not used at all.